packages/@aws-cdk/aws-dynamodb/test/test.dynamodb.ts

```diff
@@ -229,8 +229,791 @@ export = {
+        test.throws(() => table.addReadAutoScaling({
+            minimumCapacity: 50,
+            maximumCapacity: 500,
+            scalingTargetValue: 5.0,
+            scaleInCooldown: 80,
+            scaleOutCooldown: 60
+        }), Error, "scalingTargetValue for predefined metric type DynamoDBReadCapacityUtilization/"
+        + "DynamoDBWriteCapacityUtilization must be between 10 and 90");
+
+        test.done();
+    },
+
+    'when specifying Read Auto Scaling with invalid scalingTargetValue > 90 - throws Error'(test: Test) {
+        const app = new TestApp();
+        const table = new Table(app.stack, 'MyTable', {
+            tableName: 'MyTable',
+            readCapacity: 42,
+            writeCapacity: 1337
+        });
+        table.addPartitionKey('partitionKey', KeyAttributeType.String);
+        table.addSortKey('sortKey', KeyAttributeType.Binary);
+        test.throws(() => table.addReadAutoScaling({
+            minimumCapacity: 50,
+            maximumCapacity: 500,
+            scalingTargetValue: 95.0,
+            scaleInCooldown: 80,
+            scaleOutCooldown: 60
+        }), Error, "scalingTargetValue for predefined metric type DynamoDBReadCapacityUtilization/"
+        + "DynamoDBWriteCapacityUtilization must be between 10 and 90");
+
+        test.done();
+    },
+
+    'when specifying Read Auto Scaling with invalid scaleInCooldown - throws Error'(test: Test) {
+        const app = new TestApp();
+        const table = new Table(app.stack, 'MyTable', {
+            tableName: 'MyTable',
+            readCapacity: 42,
+            writeCapacity: 1337
+        });
+        table.addPartitionKey('partitionKey', KeyAttributeType.String);
+        table.addSortKey('sortKey', KeyAttributeType.Binary);
+        test.throws(() => table.addReadAutoScaling({
+            minimumCapacity: 50,
+            maximumCapacity: 500,
+            scalingTargetValue: 95.0,
+            scaleInCooldown: -5,
+            scaleOutCooldown: 60
+        }), Error, "scaleInCooldown must be greater than or equal to 0");
+
+        test.done();
+    },
+
+    'when specifying Read Auto Scaling with invalid scaleOutCooldown - throws Error'(test: Test) {
+        const app = new TestApp();
+        const table = new Table(app.stack, 'MyTable', {
+            tableName: 'MyTable',
+            readCapacity: 42,
+            writeCapacity: 1337
+        });
+        table.addPartitionKey('partitionKey', KeyAttributeType.String);
+        table.addSortKey('sortKey', KeyAttributeType.Binary);
+        test.throws(() => table.addReadAutoScaling({
+            minimumCapacity: 50,
+            maximumCapacity: 500,
+            scalingTargetValue: 95.0,
+            scaleInCooldown: 80,
+            scaleOutCooldown: -5
+        }), Error, "scaleOutCooldown must be greater than or equal to 0");
+
+        test.done();
+    },
+
+    'when specifying Read Auto Scaling with invalid maximumCapacity - throws Error'(test: Test) {
```

Comment:
For error cases, we usually start the test name with the word `"error"`, so `"error when specifying ...."`